modules/nw-route-specific-annotations.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-route-specific-annotations_{context}"]
+= Route-specific annotations
+
+Using environment variables, a router can set the default
+options for all the routes it exposes. An individual route can override some
+of these defaults by providing specific configurations in its annotations.
```

Comment:
In 4.x user cannot update environment variables directly, so better remove the description `Using environment variables,..`.